disjoint-sets.lisp

```diff
@@ -58,28 +46,40 @@ Example:
 (disjoint-sets-find sets 5)
 "
   (let ((parent (aref sets id)))
-    (if (= id parent)
-        ;; If "id" is the root, just return it.
-        id
+    (if (= id parent) id             ; If `id' is the root, return it.
         (let ((root (disjoint-sets-find sets parent)))
           ;; Path compression: point directly to the root if it's not
           ;; already the case.
           (when (/= root parent)
             (setf (aref sets id) root))
           root))))
 
-(defun disjoint-sets-union (sets id1 id2)
+(defun disjoint-sets-unify (sets id1 id2)
   "Merge two disjoint sets. Return the set representative (the root)
 
 Example:
 
-(disjoint-sets-union sets 1 2)
-=> 4 ; SETS is modified.
+(disjoint-sets-unify sets 1 2)
+=> 4 ; `sets' is modified.
```

Comment:
Is the backtick/quote convention from emacs?

Reply:
Yes, it idiomatically causes the substring between the ticks to appear highlighted differently than by `font-lock-comment-face`, using `font-lock-constant-face`. I've not surveyed enough different themes and their failure modes to have a strong opinion about whether it is worth preserving, although because it works well and is quite similar to matched delimiters, I acquired the habit of marking constants thus.

Reply:
Ok, I checked the [docs](https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html), this looks good.